### PR TITLE
PRC 166 Manage Contact Page

### PR DESCRIPTION
**Pull request by rosariodiaferia-moj**
Ticket: https://dsdmoj.atlassian.net/browse/PRC-208

Description: Manage contact placeholder page

Hmpps-Contacts-Ui-10-18-2024_06_13_PM

### CODE REVIEW

**Comment by james-baker-moj** [comment]
@rosariodiaferia-moj can we remove the tabs as well pelase? I think they are unecessary for now and we don't know when or if we'll get to them

File: integration_tests/pages/listContacts.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ export default class ListContactsPage extends Page {
     return this
   }
 
+  clickContactNamesLink(): ListContactsPage {
```
Comment (james-baker-moj) [inline integration_tests/pages/listContacts.ts:22]: does this just click the first one? I guess we are only showing a single contact?

File: server/views/pages/contacts/manage/prisonerSearchResults.njk [hunk 2]
```diff
@@ -83,7 +83,7 @@
             {% set rows = [] %}
             {% for result in results.content %}
                 {% set prisonerNameHtml %}
-                   <a href="/prisoner/{{ result.prisonerNumber }}/contacts/list/{{journey.id}}" class="govuk-link govuk-link--no-visited-state bapv-result-row">
+                   <a href="/prisoner/{{ result.prisonerNumber }}/contacts/list/{{journey.id}}" class="govuk-link govuk-link--no-visited-state bapv-result-row" data-qa="prisoner-name">
```
Comment (james-baker-moj) [inline server/views/pages/contacts/manage/prisonerSearchResults.njk:86]: should maybe be indexed or include the prisoner number in the data-qa to make sure you select the right one

File: integration_tests/pages/listContacts.ts (hunk 1 above)
Comment (rosariodiaferia-moj) [reply integration_tests/pages/listContacts.ts]: I have added the contactId to `date-qa` and and pass it as paramenter

File: server/views/pages/contacts/manage/prisonerSearchResults.njk (hunk 2 above)
Comment (rosariodiaferia-moj) [reply server/views/pages/contacts/manage/prisonerSearchResults.njk]: prisoner number in the data-qa added

**Pull request merged**